packages/sdks-tests/src/tests/blocks.spec.ts

```diff
@@ -450,9 +450,19 @@ test.describe('Blocks', () => {
     });
   });
 
-  test('nested symbols with inherit', async ({ page }) => {
+  test('nested symbols with inherit', async ({ packageName, page }) => {
     await page.goto('/nested-symbols');
 
+    // Skipping the test as v2 sdks currently don't support Slot
+    // gen1-remix and gen1-next are also skipped because React.useContext is not recognized
+    if (
+      ['react-native', 'vue2', 'vue3', 'svelte', 'qwik-city', 'gen1-remix', 'gen1-next'].includes(
```

Comment:
There are actually a lot more packages to exclude:

https://github.com/BuilderIO/builder/blob/a899abbc2aa195dc9e2ee756d72210c00912806b/packages/sdks-tests/src/tests/sdk.ts#L26-L44

```
export type PackageName =
  | 'react-native'
  | 'solid'
  | 'solid-start'
  | 'qwik-city'
  | 'next-pages-dir'
  | 'next-app-dir-client'
  | 'next-app-dir'
  | 'react'
  | 'vue2'
  | 'vue3'
  | 'nuxt3'
  | 'nuxt2'
  | 'svelte'
  | 'sveltekit'
  | 'gen1-react'
  | 'gen1-remix'
  | 'gen1-next';
```

All need to be skipped, except for `gen1-react`.

Reply:
Added all except gen1-react